LEGO1/lego/legoomni/include/legoinputmanager.h

```diff
@@ -132,6 +132,13 @@ class LegoInputManager : public MxPresenter {
 	void GetKeyboardState();
 	MxResult GetNavigationKeyStates(MxU32& p_keyFlags);
 
+	void Reset88and336()
```

Comment:
We already have this function: `DisableInputProcessing`

Reply:
Can't believe I missed that - thanks!